includes/currency_formatter.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+final class CurrencyFormatter
+{
+    private static $instance;
+
+    private static function getInstance()
+    {
+        if (self::$instance === null) {
+            self::$instance = new NumberFormatter(Locale::acceptFromHttp($_SERVER['HTTP_ACCEPT_LANGUAGE']), NumberFormatter::CURRENCY);
```

Comment:
NumberFormatter is not available without the intl PHP extension.

Can you change this part in the dockerfile?

```
# Install additional PHP extensions and dependencies
RUN apk add --no-cache libpng libpng-dev libjpeg-turbo libjpeg-turbo-dev freetype freetype-dev curl autoconf libgomp icu-dev \
    && docker-php-ext-configure gd --with-freetype --with-jpeg \
    && docker-php-ext-install -j$(nproc) gd \
    && docker-php-ext-install intl
```

And on the readme:

```
#### Baremetal

- NGINX or APACHE websever running
- PHP 7.4 or 8.0 with the following modules enabled:
    - curl
    - gd
    - imagick
    - intl
    - openssl
    - sqlite3
```

Reply:
Oops!  I didn't realise they weren't standard libraries.  This has been updated.